Resources/doc/9e-integrating_several_parsers.md

```diff
@@ -0,0 +1,45 @@
+Step 9e: Integrating several parsers together
+=============================================
+
+Purpose
+--------
+Imagine that you need to integrate ExerciseHTMLPurifierBundle parser, bbcode parser, any other parser... it's better to have several parsers where each is responsible for it's action. That's why it's better to make chain of parsers, where order is important and parsers works one after other.
+
+Usage
+-----
+
+First, configure parser classes as services. You don't have to use all parsers that this bundle provides. The main parser of this bundle is PipelineParser which provides pipeline for parser classes.
```

Comment:
``this bundle``looks weird here. It was fine in the doc of your own bundle where it was the main feature, but it is not the main feature anymore when adding it in FOSCommentBundle itself